test/data/schemas.test.ts

```diff
@@ -187,6 +192,21 @@ test('state incentives JSON files match schemas', async tap => {
           `must define city attribute on corresponding authority ${incentive.authority} for incentives with city authority type`,
         );
       }
+      if (incentive.authority_type === AuthorityType.Other) {
+        tap.hasProp(
+          incentive,
+          'geo_group',
+          `authority_type 'other' must include a geo group (id ${incentive.id})`,
+        );
+
+        const stateGroups = GEO_GROUPS_BY_STATE[stateId];
+        tap.equal(incentive.authority_type, AuthorityType.Other);
+        tap.hasProp(
+          stateGroups,
+          incentive.geo_group!,
+          `nonexistent geo_group (${stateId}, id ${incentive.id})`,
+        );
+      }
```

Comment:
Do we want to ensure that *only* Other authorities define geo_groups? Or is it intentional so that we can separate authority and eligibility even for non-Other authority types?